.github/workflows/release.yml

```diff
@@ -237,13 +237,6 @@ jobs:
               flutter_profile: production-linux-x86_64,
             }
           # - { arch: aarch64, target: aarch64-unknown-linux-gnu, os: ubuntu-20.04, extra-build-args: "", flutter_profile: production-linux-aarch64 }
```

Comment:
Might as well remove this commented out line as well then? It's in the commit history, so if we need it we can always find it again.